.codeclimate.yml

```diff
@@ -2,9 +2,21 @@ exclude_patterns:
 - "**/test/"
 - "**/androidTest/"
 - "**test"
+- "**/map/LocationRepository.kt"
 
+version: "2"
 checks:
   argument-count:
     enabled: true
     config:
-      threshold: 6
\ No newline at end of file
+      threshold: 6
+    method-lines:
+      config:
+        threshold: 35
+    similar-code:
+      enabled: false
+    identical-code:
+      enabled: false
+    file-lines:
+      config:
+        threshold: 500
```

Comment:
This should be done in a separate pr next time as it is not part of this pr's subject

Reply:
While these modifications do not directly address the subject of the PR, they are still related as the changes in the PR conflict with the "current" ruleset of codeclimate.
"current" is in quotation marks, as these are changes that I have already made in my previous PR following the group's approval. I can find them as early as commit f41ec00d0c71f235a731bf70e4ed5ad987624d49, from 2 weeks ago!

Reply:
Ok that makes sense! Actually it is simply an advise for next time, nothing crucial here.